src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -41,6 +41,11 @@ class InstPrepdisk
       def run
         return :auto if Mode.update
 
+        storage = manager.storage
+
+        storage.probed.save(Directory.logdir + "/probed-devicegraph.xml")
+        storage.staging.save(Directory.logdir + "/staging-devicegraph.xml")
+
```

Comment:
Nice if it works, so please, move it to `StorageManager#commit` to have them with all commit calls 👍